code/modules/mob/living/carbon/xenomorph/castes/hivemind/hivemind.dm

```diff
@@ -356,3 +363,30 @@
 			to_chat(parent, span_xenodanger("Your core is under attack, and low on health!"))
 		if(76 to INFINITY)
 			to_chat(parent, span_xenodanger("Your core is under attack!"))
+
+/*
+* Proc checks if we should alert the hivemind, and if it can, it does so.
+* datum/source - the atom (in this case it should be a turf) sending the crossed signal
+* atom/movable/hostile - the atom that triggered the crossed signal, in this case we're looking for a mob
+*/
+
```

Comment:
```suggestion
/**
 * Proc checks if we should alert the hivemind, and if it can, it does so.
 * datum/source - the atom (in this case it should be a turf) sending the crossed signal
 * atom/movable/hostile - the atom that triggered the crossed signal, in this case we're looking for a mob
 */
```